docs/reference/controls/itemscontrol.md

```diff
@@ -89,7 +115,61 @@ public class Crockery
 
 The view resizes horizontally, but content is hidden when it is too high. This control does not have a built-in scrollbar (unlike `ListBox`).
 
-<img src={ItemsControlScreenshot} alt="" />
+<img src={example01Screenshot} alt="" />
+
+## Example 02
+This example bind an observable collection of tile items to an items control. The ItemPanel is set to a canvas and we use a style to position the Tile within the canvas.
```

Comment:
There is a typo here, it should be "binds" not "bind". I would also avoid saying tiles and instead say `Rectangle`s since it makes it much clearer what the code is doing.